raw/mock_exam/docs/questions/question-14.adoc

```diff
@@ -0,0 +1,98 @@
+// tag::DE[]
+=== Frage 14
+**ID: Q-20-04-10**
+
+[cols="2,8,2", frame=ends, grid=rows]
+|===
+|K-Frage: 
+|Wählen Sie für jede Zeile „Richtig“ oder „Falsch“ aus.
+| 2 Punkte
+|===
+
+Geben Sie an, welche der folgenden Aussagen zu Projektzielen und Architekturzielen richtig und welche falsch sind.
+
+
+[cols="2a,2a,1, 7", frame=none, grid=none]
+|===
+
+| Richtig
+| Falsch
+|
+|
+
+| {y} 
+| {n}
+| (a)
+| Projektziele können funktionale Anforderungen sowie Qualitätsanforderungen umfassen.
+
+| {y}
+| {n}
+| (b) 
+| Architekturziele leiten sich von den Qualitätsanforderungen für das System oder Produkt ab.
+
+| {n}
+| {y}
+| (c)
+| Stakeholder aus dem Business sollten sich auf Geschäftsziele konzentrieren und sich nicht mit Architekturzielen auseinandersetzen.
+
+| {n}
+| {y}
+| (d)
+| Zur Vermeidung von Konflikten sollten Geschäfts- und Architekturziele einander nicht überlappen.
+|===
+
+// end::DE[]
+
+// tag::EN[]
+=== Question 14
+**ID: Q-20-04-10**
+
+[cols="2,8,2", frame=ends, grid=rows]
+|===
+|K-Question: 
+|Select „true“ or „false" for every line.
+| 2 points
+|===
+
+Which of the following statements regarding project goals and architectural goals is true and which is false.
+
+
+[cols="2a,2a,1, 7", frame=none, grid=none]
+|===
+
+| true
+| false
+|
+|
+
+| {y} 
+| {n}
+| (a)
+| Project Goals can include functional requirements as well as quality requirements.
+
+| {y}
+| {n}
+| (b) 
+| Architectural goals are a derived from the quality requirements for the system or product.
+
+
+| {n}
+| {y} 
+| (c) 
+| Business stakeholders should concentrate on business goals and not interfere with architectural goals.
+
+| {n}
+| {y}
+| (d)
+| To avoid conflicts business goals and architectural goals should be non- overlapping sets.
+
+|===
+
+// end::EN[]
+
+// tag::EXPLANATION[]
+Explanation:
```

Comment:
Erscheint auch in der deutschen Version

Reply:
siehe oben, explanations sind nur einsprachig möglich